hackathon_site/hardware/test_api.py

```diff
@@ -159,6 +160,58 @@ def test_hardware_quantity_success(self):
         self.assertEqual(expected_unique_hardware_counts, actual_unique_hardware_counts)
 
 
+class IncidentsListViewTestCase(SetupUserMixin, APITestCase):
```

Comment:
Also needs test cases for your two filters:
- One that filters by hardware ID. You already have two hardware instances, you'll just need to create a second order item and incident.
- One that filters by team ID. You'll have to create a second team and order to do that, which you can tie to the second order item you'll need for testing the hardware filter.